### PR TITLE
OGM-1260 Expands WildFly 10 configuration paragraph

### PR DESCRIPTION
**Pull request by DavideD**
https://hibernate.atlassian.net/browse/OGM-1260

@gsmet  Can you have a quick look at this one?

### CODE REVIEW

File: documentation/manual/src/main/asciidoc/modules/configuration.asciidoc [hunk 1]
```diff
@@ -363,15 +363,98 @@ You can download the pre-packaged module ZIP for this version of Hibernate OGM f
...
+					<artifactItem>
+						<groupId>org.hibernate</groupId>
+						<artifactId>hibernate-orm-modules</artifactId>
+						<version>{hibernate-orm-version}</version>
+						<classifier>{hibernate-wildfly-classifier}</classifier>
+						<type>zip</type>
+						<overWrite>false</overWrite>
+						<outputDirectory>
+                            ${project.build.directory}/wildfly-{wildfly-version}/modules
+                        </outputDirectory>
```
Comment (gsmet) [inline documentation/manual/src/main/asciidoc/modules/configuration.asciidoc]: The indentation looks weird here.

File: documentation/manual/src/main/asciidoc/modules/configuration.asciidoc [hunk 2]
```diff
@@ -363,15 +363,98 @@ You can download the pre-packaged module ZIP for this version of Hibernate OGM f
...
+					<artifactItem>
+						<groupId>org.hibernate.ogm</groupId>
+						<artifactId>hibernate-ogm-modules</artifactId>
+						<classifier>{hibernate-wildfly-classifier}</classifier>
+						<version>{hibernate-ogm-version}</version>
+						<type>zip</type>
+						<overWrite>false</overWrite>
+						<outputDirectory>
+                            ${project.build.directory}/wildfly-{wildfly-version}/modules
+                        </outputDirectory>
```
Comment (gsmet) [inline documentation/manual/src/main/asciidoc/modules/configuration.asciidoc]: Same here.

File: documentation/manual/src/main/asciidoc/modules/configuration.asciidoc [hunk 3]
```diff
@@ -363,15 +363,98 @@ You can download the pre-packaged module ZIP for this version of Hibernate OGM f
...
 The modules included are:
 
 * _org.hibernate.ogm_, the core Hibernate OGM library.
 * _org.hibernate.ogm.<%DATASTORE%>_, one module for each datastore, with _<%DATASTORE%>_ being one of _infinispan_, _mongodb_ etc.
+* _org.hibernate.orm_, the Hibernate ORM libraries.
 * Several shared dependencies such as _org.hibernate.hql:<%VERSION%>_ (containing the query parser) and others
 
 The module slot to use for Hibernate OGM {hibernate-ogm-version} is `{hibernate-ogm-module-slot}` as the format of the slot name does not include the "micro" part of the project version.
 
+To uprade the Hibernate ORM version you will need to add the following property to your `persistence.xml`:
```
Comment (gsmet) [inline documentation/manual/src/main/asciidoc/modules/configuration.asciidoc]: s/uprade/upgrade/

**Comment by DavideD** [comment]
Remarks applied

**Comment by DavideD** [comment]
Thanks

**Comment by gsmet** [comment]
Merged, thanks!

**Pull request merged**